apps/pg_subscriber/lib/pg_subscriptor/handler.ex

```diff
@@ -134,6 +140,31 @@ defmodule PgSubscriber.Handler do
     Logger.info(tuple_data: tuple_data)
   end
 
+  defp handle_delete!(body) do
+    Logger.info("Got DELETE msg")
+    <<_relation_oid::32, tuple_type::8, rest::binary>> = body
+    {:ok, tuple_data, <<>>} = TupleData.get_tuple_data(rest)
+
+    Logger.info(tuple_type: <<tuple_type>>)
+    Logger.info(tuple_data: tuple_data)
+  end
+
+  defp handle_truncate!(body) do
+    Logger.info("Got TRUNCATE msg")
+    <<relations_num::32, _options::8, rest::binary-size(4 * relations_num)>> = body
+    relation_oids = get_relations_oid(relations_num, rest)
+    Logger.info(relation_oids: relation_oids)
+  end
+
+  defp get_relations_oid(0, _) do
+    []
+  end
+
+  defp get_relations_oid(relations_num, msg) do
```

Comment:
[nitpick]
`get_relations_oids` maybe?

And I think we should start to use one name for this binary parameter, maybe? We use `body`, `msg`, `rest`, I used `data` (as in binary data)... I mean, sure, sometimes it makes sense, but for example, now I feel like the function expects the whole PG message...

Reply:
Yeah, I am mixing it up.. I suggest calling `message`/`msg` the whole replication message (to follow Postgres style). And call unparsed (binary) part of a message `data`. What do you think?